vendor/bat-native-ads/src/bat/ads/pref_names.cc

```diff
@@ -71,8 +71,8 @@ const char kHasMigratedRewardsState[] =
     "brave.brave_ads.migrated.rewards_state";
 
 // Stores hash
-const char kConfirmationsHash[] = "brave.brave_ads.confirmations.hash";
-const char kClientHash[] = "brave.brave_ads.client.hash";
```

Comment:
We must migrate client state otherwise the hash will not match due to migration of snake_case to camelCase